src/fidesops/api/v1/endpoints/oauth_endpoints.py

```diff
@@ -76,7 +76,9 @@ async def acquire_access_token(
     else:
         raise AuthenticationFailure(detail="Authentication Failure")
 
-    client_detail = ClientDetail.get(db, object_id=client_id, config=config)
+    client_detail = ClientDetail.get(
+        db, object_id=client_id, config=config, scopes=SCOPE_REGISTRY
```

Comment:
From an organizational perspective, this is confusing, in that these scopes are only used if the client is the root client, otherwise, we use their actual associated scopes, but just looking at this line, that is not necessarily obvious. I'd at least add a code comment here explaining that.

Reply:
good point, I'd love to rename `scopes` to `root_scopes` or `all_scopes` or something in future, though that'll require more fideslib changes. For now, let's get this working, and I'll update with a comment

Reply:
thank you for adding!